src/Security/MemberAuthenticator/ChangePasswordHandler.php

```diff
@@ -287,6 +289,10 @@ public function doChangePassword(array $data, $form)
             return $this->redirect($backURL);
         }
 
+        $backURL = Security::config()->get('default_reset_password_dest');
+        if ($backURL) {
```

Comment:
Not using strict type checking, but depending on the PHP setting, it might. Hence the explicit false down in Security.

Reply:
`null` is preferred for unset values. We've had this issue with get_by_id() returning false for non-matches as well.

Reply:
Sure. Could you please give me a heads up to rework this? I've got too much going to keep track of everything :D